extensions/LukeManiaStudios/lmsutils.js

```diff
@@ -1399,65 +1280,33 @@
     }
 
     screenReporter(args) {
-      if (args.DROPDOWN === 'width') {
-        return screen.width;
-      }
-      if (args.DROPDOWN === 'height') {
-        return screen.height;
-      }
+      if (args.DROPDOWN === 'width') return screen.width;
+      if (args.DROPDOWN === 'height') return screen.height;

```

Comment:
a detail that should've been caught earlier: this and the block below need to return *something* when args.DROPDOWN is none of those